x-pack/qa/full-cluster-restart/src/javaRestTest/java/org/elasticsearch/xpack/restart/MLModelDeploymentFullClusterRestartIT.java

```diff
@@ -139,11 +138,6 @@ private void waitForDeploymentStarted(String modelId) throws Exception {
 
     private void assertInfer(String modelId) throws IOException {
         Response inference = infer("my words", modelId);
-        assertThat(EntityUtils.toString(inference.getEntity()), equalTo("{\"predicted_value\":[[1.0,1.0]]}"));
```

Comment:
Previously the upgrade tests would test the deprecated route in and the new route. The deprecated route is now removed